docs/source/session.rst

```diff
@@ -28,6 +28,7 @@ Snowpark Session
       Session.create_async_job
       Session.create_dataframe
       Session.flatten
+      Session.generator
```

Comment:
It makes me feel the auto generation should be prioritized after GA :)

Reply:
Exactly.